src/modules/users/CheckPWQuality.h

```diff
@@ -55,9 +57,19 @@ class PasswordCheck
         return m_accept( s ) ? QString() : m_message();
     }
 
+    /** Retturns a unique identifier of the check.
+     *  Consumers of this class may use this for special handling of specific checks.
+     *  Currently the only handled option is 'libpwquality'.
```

Comment:
"some families of checks", "handled option" -> "family that is handled specially", `libpwquality` should match the name of the family / Id from CheckId.